.github/workflows/e2e-test.yml

```diff
@@ -0,0 +1,106 @@
+name: End-To-End Testing
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  ubuntu-16:
+    runs-on: ubuntu-16.04
+    name: 'Ubuntu 16.04'
+    steps:
+      - run: sudo systemctl start mysql.service && sudo apt-get update && sudo apt-get install -y nginx
+        name: 'Setting up required services'
+      - run: 'echo -e "127.0.0.1 cli-testing.ghost.org\n" | sudo tee -a /etc/hosts'
+        name: 'Updating hosts file'
+      - uses: actions/checkout@v2
+        with:
+          path: 'cli'
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - run: cd cli && yarn && sudo yarn link --link-folder /bin/
```

Comment:
Is there a reason we need to use link? What's wrong with running `./cli/bin/ghost ...`?

Reply:
that could work too - wanted to try and make it as close to real-world as possible though

Reply:
Gotcha. Main reason I was asking is that seems to be the only difference between the 3 OSes - if they follow the same template, matrixing will be super easy

Reply:
I'll test and see if /usr/bin works on ubuntu 16/18.

Reply:
yup looks like `/usr/bin/` works consistently across all three - updated to just use that